### PR TITLE
Add Statistical Data Sets format

### PR DESCRIPTION
**Pull request by bevanloon**
Add schema and examples for Statistical Data Sets.

[Trello](https://trello.com/c/3OZ36DPk)

Related pull request in [government-frontend](https://github.com/alphagov/government-frontend/pull/199)

Paired with @fofr and @tvararu

### CODE REVIEW

File: formats/statistical_data_set/frontend/examples/statistical_data_set_withdrawn.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{
+  "base_path": "/government/statistical-data-sets/tsgb10-withdrawn",
```
Comment (fofr) [inline formats/statistical_data_set/frontend/examples/statistical_data_set_withdrawn.json]: I'd prefer a real withdrawn example. This helps with:
* like by like comparisons when building or iterating the frontend
* confirming or refuting assumptions made when building the frontend for the first example

**Comment by fofr** [comment]
Is there an example of a political statistical data set?

File: formats/statistical_data_set/publisher/details.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+         }
+       }
+     },
+     "government": {
+       "$ref": "#/definitions/government"
+     },
+     "political": {
+       "$ref": "#/definitions/political"
+     },
+     "emphasised_organisations": {
```
Comment (fofr) [inline formats/statistical_data_set/publisher/details.json]: Perhaps `tags` too.

**Comment by bevanloon** [comment]
There are no political Statistical Data Sets from what I can see. The spike week document notes that political is always false for Statistical Data Sets.

**Comment by fofr** [comment]
@bevanloon The UI suggests that this might not always be the case? Is it possible to create a political one within Whitehall?

File: formats/statistical_data_set/frontend/examples/statistical_data_set_withdrawn.json (hunk 1 above)
Comment (bevanloon) [inline formats/statistical_data_set/frontend/examples/statistical_data_set_withdrawn.json]: Real example added and squashed into commit 1b1a946

**Comment by bevanloon** [comment]
A political Statistical Data Set example has been created as per commit 5792818. Since there are no political Statistical Data Sets on gov.uk, a fake example was created.

File: formats/statistical_data_set/publisher/details.json (hunk 2 above)
Comment (bevanloon) [inline formats/statistical_data_set/publisher/details.json]: Sounds good - should we get some cards added to trello for it?

Comment (bevanloon) [inline formats/statistical_data_set/publisher/details.json]: [Trello ticket](https://trello.com/c/g2zCKSFQ/520-refactor-emphasised-organisations-and-tags-in-details-json-in-govuk-content-schemas) created.

**Comment by fofr** [comment]
Changes look good to me :+1:. Leave the merging/final review to someone else as I helped work on this.

**Pull request merged**